.travis.yml

```diff
@@ -2,15 +2,16 @@ language: scala
 sudo: required
 jdk:
   - oraclejdk8
+addons:
+  apt:
+    packages:
+      - oracle-java8-set-default
```

Comment:
Hmm, this looks weird, I'm guessing there's competing JDK's caused by `.deb` dependencies. Do we have some docs pointing to this approach?